src/styles/blocks/rating.scss

```diff
@@ -0,0 +1,13 @@
+.rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-inline: $standart-inline-pad;
```

Comment:
The variable name '$standart-inline-pad' contains a typo. It should be '$standard-inline-pad' to ensure correct spelling and consistency across your styles.